rts/at/tx/src/main/java/org/jboss/jbossts/star/resource/RESTRecord.java

```diff
@@ -212,8 +211,7 @@ public int topLevelPrepare() {
         if (prepareURI == null || txId == null)
```

Comment:
This one needs braces as well

Reply:
Actually this isn't in our style rules so I guess keep it as it is.

But should we include it in our checkstyle rules?

Reply:
I added the `LITERAL_IF` token (https://checkstyle.sourceforge.io/config_blocks.html#LeftCurly) to the ruleset and it didn't report the missing curly:

```
         <module name="LeftCurly">
             <property name="option" value="eol"/>
-            <property name="tokens" value="CLASS_DEF,CTOR_DEF,INTERFACE_DEF,METHOD_DEF,LITERAL_SWITCH,STATIC_INIT"/>
+            <property name="tokens" value="LITERAL_IF,CLASS_DEF,CTOR_DEF,INTERFACE_DEF,METHOD_DEF,LITERAL_SWITCH,STATIC_INIT"/>
         </module>
```

Reply:
I changed only the try-catch blocks. I wouldn't change the checkstyle configuration because we would need to change  LITERAL_TRY  LITERAL_IF LITERAL_ELSE and so on and I am afraid we would appear too strict. I'd rather contributors were coherent with the module code style/format.